cfme/infrastructure/virtual_machines.py

```diff
@@ -685,7 +685,7 @@ def delete(self, cancel=False):
             if not cancel:
                 # TODO: test this in test_snapshot_crud, just assert_no_error here
                 view.flash.assert_message(
-                    "Delete Snapshot initiated for 1 VM and Instance from the ManageIQ Database"
+                    "Delete Snapshot initiated for 1 VM and Instance from the CFME Database"
```

Comment:
This should likely use `appliance.product_name` property in order to be compatible with upstream and downstream. This was likely left as MIQ by accident downstream, and has been 'fixed' in the downstream images. Once we test upstream images, this will still be 'MIQ' in the appliance under test.

Reply:
I was thinking something similar. Will fix.

Also, there's another issue here I need to fix before RFR, with `test_create_snapshot_via_ae`.